wiki/People/Beatmap_Nominators/Rules/en.md

```diff
@@ -8,46 +8,62 @@ Beatmap Nominators possess the tools to nominate beatmaps and shape the world of
 
 ## Rules
```

Comment:
i dont think 2 "rules" headers are necessary. the page is called rules already, so this text about what rules are doesnt need separation